docs/workgroup/charter.rst

```diff
@@ -0,0 +1,136 @@
+===================================
+CPython Documentation Working Group
+===================================
+
+Purpose & Common Goals
+----------------------
+
+This workgroup will support efforts to improve
+and maintain CPython's documentation through curation,
+internationalization, and special projects. Its focus will also include
+tutorials, how-to documents, the devguide and other documentation in support
+of the PSF's mission to expand the global Python userbase.
+Quality and effectiveness will be key goals as we build a
+diverse team of community members, documentarians, and core developers.
+
+Things the workgroup could do:
+
+- Develop the governance model for docs
+- Be the steering council for docs
+- Be the authors of doc guidelines
+- Be the authors/editors of docs
+- Request grant funding from PSF, if needed
+- Form an editorial board for docs including core devs, educators, and
+  documentarians in the Python community
```

Comment:
Given that the group has started, this list feels out of place -- perhaps it should move to a TODO document? A speculative list seems off for a terms of reference.

Reply:
I agree with that logic, but that would be a pretty significant content-relevant change in what I scoped to be a copyediting-focused PR with minimal substantive changes to the actual content/meaning.

Reply:
I think it's fine for a charter to include a list of possible work. It helps makes the details more concrete.